changelogs/fragments/improve_test_coverage.yaml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
```

Comment:
I'd argue this is `trivial`... test and coverage updates aren't generally something I think needs to be called out in a changelog.